CHANGELOG.md

```diff
@@ -8,6 +8,43 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.3.1] - 2023-08-23
+
+### Added
+* Count datastore queries involved in Check resolution metadata ([#880](https://github.com/openfga/openfga/pull/880))
+
+  OpenFGA request logs will now include a field `datastore_query_count` that shows how many queries were involved in a single Check resolution.
```

Comment:
It's not just logs right? It's also traces and metrics?

Reply:
Ah it's below LOL

Reply:
We don't have a `datastore_query_count` label per metric for Check, those are bucketed. But we do have it in traces as well.

Reply:
Updated